packages/riverpod_generator/test/annotated_test.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:riverpod/riverpod.dart';
+import 'package:test/test.dart';
+
+import 'integration/annotated.dart';
+
+void main() {
+  test('Applied meta annotations to provider', () {
+    //TODO (SunlightBro): check if annotations are correctly applied.
+    expect(functionalProvider, isA<AutoDisposeProvider<String>>());
```

Comment:
Have to check if there is a Matcher for checking annotations in test, or write my own.
Or do you have a better way to test for annotations?

Reply:
Use parseString/file or resolveFile from analyzer